squad/frontend/static/squad/build.js

```diff
@@ -19,6 +20,15 @@ app.controller(
     ]
 )
 
+app.controller(
+    'BuildComparisonController',
+    [
+        '$scope',
+        '$window',
+        BuildComparisonController
+    ]
+)
+
```

Comment:
this should probably be added to a separate flle. build.js is supposed to be included in pages where a single build is shown; all the rest of the code imported in build.js would not be useful in the build list

Reply:
I think I'll place it in here `squad/frontend/static/squad/compare.js`